esp/smc/SMCLib/TpCommon.cpp

```diff
@@ -183,21 +183,13 @@ static SecAccessFlags getDropZoneScopePermissions(IEspContext& context, const IP
         throw makeStringException(ECLWATCH_INVALID_CLUSTER_NAME, "getDropZoneScopePermissions(): DropZone path must be specified.");
 
     //If the dropZonePath is an absolute path, change it to a relative path.
```

Comment:
looks the code below breaks this rule ?
Doesn't it need to conditionally call getRelativePath ? (if isAbsolute..)

Reply:
Will check if isAbsolute ...